javascript/ql/src/semmle/javascript/Functions.qll

```diff
@@ -158,6 +158,51 @@ class Function extends @function, Parameterized, TypeParameterized, StmtContaine
     result = getAReturnStmt().getExpr()
   }
 
+  /** 
+   * Gets a return from a function which has undefined value (i.e. implicit returns
+   * and returns w/o expressions).
```

Comment:
No informal or non-standard abbreviations, please, they can be difficult to understand for us non-native speakers.